### PR TITLE
Result location elements built from locationBase type

### PR DESCRIPTION
**Pull request by allthatilk**
Making activity schema more DRY by extracting location element before dealing with new added location element at results/indicator/baseline/

### CODE REVIEW

File: iati-activities-schema.xsd [hunk 1]
```diff
@@ -1451,7 +1451,7 @@
     </xsd:complexType>
   </xsd:element>
 
-  <xsd:element name="location">
+  <xsd:element name="location" type="locationFull">
     <xsd:annotation>
       <xsd:documentation xml:lang="en">
         The sub-national geographical identification of the target
```
Comment (dalepotter) [inline iati-activities-schema.xsd]: As we're changing the entire element, let's fix the wrapping!!

File: iati-activities-schema.xsd [hunk 2]
```diff
@@ -2867,5 +2582,270 @@
...
 
+<!-- locationBase provides only the ref attribute for location -->
+  <xsd:complexType name="locationBase">
+    <xsd:sequence>
+      <xsd:any namespace="##other" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>
+    </xsd:sequence>
+    <xsd:attribute name="ref" use="optional" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          A cross-reference to the internal reference assigned
```
Comment (dalepotter) [inline iati-activities-schema.xsd]: pesky hard wrapping again ;-)

**Review by dalepotter — CHANGES_REQUESTED** [review]
Good work on abstracting this. Just a couple comments on line wrapping.

File: iati-activities-schema.xsd (hunk 2 above)
Comment (allthatilk) [reply iati-activities-schema.xsd]: Darn the hard wrapping! T_T

File: iati-activities-schema.xsd [hunk 3]
```diff
@@ -2867,5 +2579,269 @@
...
+
+<!-- locationFull extends locationBase to include all sub-elements for location at the iati-activity level -->
+  <xsd:complexType name="locationFull">
+    <xsd:complexContent>
+      <xsd:extension base="locationBase">
+        <xsd:sequence>
+          <xsd:element name="location-reach" minOccurs="0" maxOccurs="1">
+            <xsd:annotation>
+              <xsd:documentation xml:lang="en">
+                 Does this location describe where the activity takes
```
Comment (dalepotter) [inline iati-activities-schema.xsd]: Dangit - the hard wrapping is here too. 

I'd suggest any new lines added should not have hard wrapping. Once this is fixed for this and new lines below, it's good to go!

**Review by dalepotter — CHANGES_REQUESTED** [review]

File: iati-activities-schema.xsd (hunk 3 above)
Comment (allthatilk) [reply iati-activities-schema.xsd]: I have nothing good to say about the irritation this removing hard wrapping from old data is causing.

**Review by dalepotter — APPROVED** [review]

**Pull request merged**